examples/seismic/tti/tti_example.py

```diff
@@ -64,20 +58,24 @@ def run(dimensions=(50, 50, 50), spacing=(20.0, 20.0, 20.0), tn=250.0,
                         type=int, help="Time order of the simulation")
     parser.add_argument("-so", "--space_order", default=4,
                         type=int, help="Space order of the simulation")
-    parser.add_argument("--nbpml", default=10,
+    parser.add_argument("--nbpml", default=40,
```

Comment:
remind me why is that?

Reply:
Even if 10 doesn't break anything, ABC (pml) coefficients are designed for 10, and on top of that, kinda make sure we "always" run at least the full model for high order (so 8 with shifted is already out with nbpml=10)